apps/hex2mif.py

```diff
@@ -0,0 +1,125 @@
+'''
+This script helps to convert hex instructions .mif files.
+To run it, use "python hex2mif.py input.txt/sv output.mif"
+
+you need to export the binary instruction as hexadecimal text file.
+But for any kind of input files, it should work fine as long as
+it uses the following format:
+DEADBEEF
+BAADF00D
+'''
+
+from asyncio.windows_events import NULL
+import sys
+
+comment_msg='''-- http://srecord.sourceforge.net/

```

Comment:
This comment is confusing.
Please explain.
And print that "Generated automatically by python script hex2mif.py - Matan Eshel"